src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts

```diff
@@ -124,6 +126,9 @@ class ZWaveJSNodeConfig extends SubscribeMixin(LitElement) {
     }
 
     const device = this._device!;
+    const endpoints = Object.values(this._config)
+      .map((item) => item.endpoint)
+      .filter((value, index, array) => array.indexOf(value) === index);
```

Comment:
What are you trying to do here?

Reply:
Use the `groupBy` helper <https://github.com/home-assistant/frontend/blob/dev/src/common/util/group-by.ts#L1> to group the config by endpoint, then we don't have to filter it everytime